_episodes/05-preprocessor.md

```diff
@@ -244,55 +239,64 @@ specific preprocessor which should be applied.
 >> Here is one solution to complete the challenge above using  two different
 >> preprocessors:
 >>
->>```yaml
->>
->> datasets:
->>   - {dataset: UKESM1-0-LL, project: CMIP6, exp: historical,
->>      ensemble: r1i1p1f2}  # single dataset as an example
->>
->> preprocessors:
->>   prep_map:
->>     regrid:  # apply the preprocessor to regrid
->>       target_grid: 1x1  # target resolution
->>       scheme: linear  # how to interpolate for regridding
+>>```diff
+>> --- data/recipe_example.yml
+>> +++ data/recipe_example_multi_preprocessors.yml
+>> @@ -20,23 +20,37 @@
+>>      - ukesm
 >>
->>   prep_map_land:
->>     custom_order: true  # ensure that given order of preprocessing is followed
->>     mask_landsea:    # apply a mask
->>       mask_out: sea   # mask out sea grid cells
->>     regrid:  # now apply the preprocessor to regrid
->>       target_grid: 1x1  # target resolution
->>       scheme: linear  # how to interpolate for regridding
+>>  datasets:
+>> -  - {dataset: HadGEM2-ES, project: CMIP5, exp: historical, mip: Omon, ensemble: r1i1p1, start_year: 1900, end_year: 2000}
+>> +  - {dataset: HadGEM2-ES, project: CMIP5, exp: historical, mip: Amon, ensemble: r1i1p1, start_year: 2000, end_year: 2005}
```

Comment:
Can we keep the end year the same as original recipe? 
Start year still needs to be set to >1958 due to missing data so I suggest we use 1970.
```suggestion
>> +  - {dataset: HadGEM2-ES, project: CMIP5, exp: historical, mip: Amon, ensemble: r1i1p1, start_year: 1970, end_year: 2000}
```

Reply:
The variable `thetao` in HadGEM2-ES starts in 1999 and the variable `pr` in GPCP-SG starts in 1979. For consistency in all of the examples/recipes in this episode, I selected 2000-2005 as start/end year.

Reply:
OK, consistency within episode is good